hack/update-crds.sh

```diff
@@ -5,5 +5,5 @@ set -o nounset
 set -o pipefail
 
 echo "Generating CRDs With controller-gen"
-GO11MODULE=on go install sigs.k8s.io/controller-tools/cmd/controller-gen
-controller-gen crd paths=./pkg/apis/cluster/... output:crd:dir=./deploy/crds
+GO11MODULE=on go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.8.0
+cd staging/src/github.com/clusterpedia-io/api && controller-gen crd paths=./cluster/... output:crd:dir=../../../../../deploy/crds
```

Comment:
Also use `REPO_ROOT` and `STAGING_REPO_ROOT` to avoid `cd taging/src/github.com/clusterpedia-io/api` and `..`

Reply:
done